### ISSUE TITLE
Scrolling issues with JMM Desktop 3.4.4.3

### ISSUE THREAD

**Issue by xsystematic** [japanesemediamanager/jmmclient#241]
I can no longer use the mouse wheel to scroll up or down while viewing series info or episodes when checking my collection. I can still use it when scrolling through my collection though.

Another issue I have noticed is that while viewing any anime, on the series info tab,when I left click on numerous areas within the tab itself (such as expanding overview or auto link on the tvdb option) it will cause the page to scroll down just enough to hide the toolbar with the edit and move series options.

I failed to replicate these issues with JMM Desktop 3.4.3.1 and the beta 3.4.4.1 has been removed so I'm unable to test it.

**Comment by AreFinstad** [comment]
can confirm, does not work at Detailed series page under Collection but does work at pinned.
did work last week i think..... tho i havent really used JMM much since the last beta release.

**Comment by ElementalCrisis** [comment]
I think your second issue is related to something I've noticed, the overview seems to expand when you click on anything else that expands.

**Comment by ElementalCrisis** [comment]
Figured it out, funny enough this was my first guess but I tried everything else before trying this. The ScrollViewer that encompassed the Series Info part was causing problems. Removed it and added a ScrollViewer to the sticky tab section of MainWindow. Now series in collection scroll properly and pinned series still have a scroll bar. This should fix both of your problems @xsystematic and my problem I thought was a problem and related was neither, I'm just dumb.

**Comment by ElementalCrisis** [comment]
Re-reading the main post I realize I missed the part about the episodes tab. I gave it a go but haven't been able to fix it yet. I'll keep going at it but it might be a little outside my skill level.

**Comment by xsystematic** [comment]
Thanks for looking into it, not a huge problem so there's no rush to get it fixed

**Comment by ElementalCrisis** [comment]
No worries, I like doing this kind of thing and it helps me learn more.

**Comment by ElementalCrisis** [comment]
Was getting no where so I started from the beginning undoing all my changes and I managed to find the root of the problem. In MainWindow.xaml it's the added ScrollViewer and the change to the ContentControl named ccDetail. This bug was created by a bug I tried to fix in #232, sorry about that.  My bug was minor so if the fix for it needs to be rolled back to fix this bug I'm all for it as this is a bigger issue. In the meantime I'll see if I can find a solution to fix both problems.
